models/public/ssh/ssh.md

```diff
@@ -0,0 +1,101 @@
+# ssh
```

Comment:
Looks like it's a reimplementation of the original Caffe model, so it ought to be called `ssh-mxnet`.

Reply:
Renamed